Binary/Loader/FileSystemLoader.php

```diff
@@ -38,30 +38,48 @@ class FileSystemLoader implements LoaderInterface
     /**
      * @param MimeTypeGuesserInterface  $mimeGuesser
      * @param ExtensionGuesserInterface $extensionGuesser
-     * @param string[]                  $dataRoots
-     * @param LocatorInterface          $locator
+     * @param string[]|LocatorInterface $locator
      */
```

Comment:
I think https://github.com/liip/LiipImagineBundle/pull/963/commits/23e79a7f41321010a2f8a38a00f49916ca64704a should do the trick. :-)